bve/src/parse/mesh/instructions/creation.rs

```diff
@@ -23,11 +23,34 @@ pub(in crate::parse::mesh::instructions) fn b3d_to_csv_syntax(input: &str) -> St
     p
 }
 
+enum DeserializeInstruction {
```

Comment:
Is this really an instruction? I don't feel like it is

Reply:
Facc, that got munged in a set of renames I did, fixing.